locales/en/common.json

```
@@ -5,42 +5,42 @@
 
   "MMASK_LEAD_HEADER": "Install MetaMask",
   "MMASK_LEAD_EXPLANATION": "To use this web app, your browser needs to have MetaMask.",
-  "MMASK_MODAL_MOBILE_BODY": "Open or Install the MetaMask App",
-  "MMASK_MODAL_MOBILE_CTA": "Open the MetaMask App",
+  "MMASK_MODAL_MOBILE_BODY": "Open or Install the MetaMask app.",
+  "MMASK_MODAL_MOBILE_CTA": "Open the MetaMask app.",
   "MMASK_MODAL_BODY": "Install the browser extension for MetaMask",
   "MMASK_MODAL_CTA": "Install MetaMask",
 
   "MINT_YOUR_KAMON_HEADING": "Mint your Kamon",
-  "MINT_YOUR_KAMON_EXPLANATION": "The Kamon NFT proves your membership of the HENKAKU community.",
-  "MINT_YOUR_KAMON_DETAILS": "You can mint your Kamon NFT for 1000 $HENKAKU.",
-  "MINT_YOUR_KAMON_DETAILS_NOTE": "(Plus gas)",
+  "MINT_YOUR_KAMON_EXPLANATION": "The Kamon NFT proves that you are a member of the Henkaku community.",
+  "MINT_YOUR_KAMON_DETAILS": "You can mint a personalized Kamon NFT for 1000 $HENKAKU.",
+  "MINT_YOUR_KAMON_DETAILS_NOTE": "(Plus Gas Fees)",
```

This is just a note that will appear after the above, so why have this in title case?

Suggestion:

```suggestion
  "MINT_YOUR_KAMON_DETAILS_NOTE": "(Plus gas fees)",
```